docs/admin/Offline_System_Certificate_Renewal.md

```diff
@@ -120,6 +110,17 @@ It is recommended to run the following steps to ensure that `CS.cfg` and NSS dat
     ````
     **OR**
 
+    If your **admin cert is expired** and TLS/SSL is configured for LDAP [WORK IN PROGRESS]:
```

Comment:
If this scenario is not yet supported by the current code, it probably should not be included in the documentation, or be moved into a wiki page instead.

Reply:
okie :)